set.mm

```diff
@@ -408952,116 +408976,126 @@ function instead of the edges themselves (see also ~ nbgrval ).
-    $( The neighbors of a vertex in a graph are a subset of all vertices of the
-       graph.  (Contributed by Alexander van der Vekens, 12-Oct-2017.)
-       (Revised by AV, 26-Oct-2020.) $)
-    nbgrssvtx $p |- ( G e. W -> ( G NeighbVtx N ) C_ V ) $=
-      ( vn wcel cnbgr co cv nbgrisvtx ex ssrdv ) ADGZFABHIZCNFJZOGPCGABPCDEKLM
-      $.
-
-    $d G e n v $.
-    $( A vertex in a graph is not a neighbor of itself.  (Contributed by by AV,
-       3-Nov-2020.)  (Revised by AV, 21-Mar-2021.) $)
+    $d V e n v $.
+    nbgrnself.v $e |- V = ( Vtx ` G ) $.
+    $( A vertex in a graph is not a neighbor of itself.  (Contributed by
+       Alexander van der Vekens, 12-Oct-2017.)  (Revised by AV, 3-Nov-2020.)
+       (Revised by AV, 21-Mar-2021.) $)
     nbgrnself $p |- A. v e. V v e/ ( G NeighbVtx v ) $=
       ( vn ve cv cnbgr co wnel wcel cpr wss cedg cfv wrex csn cdif crab wa wceq
       neldifsnd intnanrd df-nel preq2 sseq1d rexbidv elrab xchbinx sylibr eqidd
       wn eqid nbgrval neleq12d mpbird rgen ) AGZBURHIZJZACURCKZUTURUREGZLZFGZMZ
       FBNOZPZECURQRZSZJZVAURVHKZURURLZVDMZFVFPZTZULVJVAVKVNVAURCUBUCVJURVIKVOUR
       VIUDVGVNEURVHVBURUAZVEVMFVFVPVCVLVDVBURURUEUFUGUHUIUJVAURURUSVIVAURUKFEVF
       BURCDVFUMUNUOUPUQ $.
+  $}
 
-    $( A vertex in a simple graph is not a neighbor of itself.  (Contributed by
-       Alexander van der Vekens, 12-Oct-2017.)  (Revised by AV, 27-Oct-2020.)
-       (Proof shortened by AV, 3-Nov-2020.) $)
-    usgrnbnself $p |- ( G e. USGraph -> A. v e. V v e/ ( G NeighbVtx v ) ) $=
-      ( cv cnbgr co wnel wral cusgr wcel nbgrnself a1i ) AEZBNFGHACIBJKABCDLM
-      $.
+  ${
+    $d G v $.  $d X v $.
+    $( A class ` X ` is not a neighbor of itself (whether it is a vertex or
+       not).  (Contributed by Alexander van der Vekens, 12-Oct-2017.)  (Revised
+       by AV, 3-Nov-2020.)  (Revised by AV, 12-Feb-2022.) $)
+    nbgrnself2 $p |- X e/ ( G NeighbVtx X ) $=
+      ( vv cvtx cfv wcel cnbgr co wnel cv wceq id oveq2 neleq12d eqid nbgrnself
+      vtoclri wn nbgrisvtx con3i df-nel sylibr pm2.61i ) BADEZFZBABGHZIZCJZAUHG
+      HZIUGCBUDUHBKZUHBUIUFUJLUHBAGMNCAUDUDOZPQUERBUFFZRUGULUEABBUDUKSTBUFUAUBU
+      C $.
+
+    $d V v $.
+    nbgrssovtx.v $e |- V = ( Vtx ` G ) $.
+    $( The neighbors of a vertex ` X ` are a subset of all vertices except the
+       vertex itself.  Stronger version of ~ nbgrssvtx .  (Contributed by
+       Alexander van der Vekens, 13-Jul-2018.)  (Revised by AV, 3-Nov-2020.)
+       (Revised by AV, 12-Feb-2022.) $)
+    nbgrssovtx $p |- ( G NeighbVtx X ) C_ ( V \ { X } ) $=
+      ( vv cnbgr co csn cdif cv wcel nbgrisvtx wceq wn nbgrnself2 df-nel neleq1
+      wne wnel syl5bbr mpbiri necon2ai eldifsn sylanbrc ssriv ) EACFGZBCHIZEJZU
+      FKZUHBKUHCRUHUGKACUHBDLUIUHCUHCMZUINZCUFSZACOUKUHUFSUJULUHUFPUHCUFQTUAUBU
+      HBCUCUDUE $.
+
+    $( The neighbors of a vertex ` X ` are a subset of all vertices except the
+       vertex itself and a vertex which is not a neighbor.  (Contributed by
```

Comment:
vertex which is not --> class which is not

Reply:
Done